employee-rostering-frontend/translations/en/ShiftRosterPage.yaml

```diff
@@ -0,0 +1,9 @@
+---
+createShift: Create Shift
+noSpots: 
+  <0>There are no Spots</0>
+  <1>
+    The current tenant have no Spots. You need at least one Spot to see the Shift Roster.
+    You can add a Spot in the Spots page.
```

Comment:
For consistency, surround Spots with quotations in "You can add a Spot in the "Spots" page" as in Employee and AvailabilityRoster pages.